x/ibc/20-transfer/types/msgs.go

```diff
@@ -76,14 +76,14 @@ func (msg MsgTransfer) GetSigners() []sdk.AccAddress {
 }
 
 type MsgRecvPacket struct {
-	Packet channelexported.PacketI `json:"packet" yaml:"packet"`
-	Proofs []commitment.Proof      `json:"proofs" yaml:"proofs"`
-	Height uint64                  `json:"height" yaml:"height"`
-	Signer sdk.AccAddress          `json:"signer" yaml:"signer"`
+	Packet channelexported.PacketDataI `json:"packet" yaml:"packet"`
```

Comment:
why is this a packet data instead of a packet?

Reply:
Agreed, the whole packet needs to be passed here, right?